src/SFML/Graphics/Text.cpp

```diff
@@ -48,7 +48,7 @@ namespace
     }
 
     // Add a glyph quad to the vertex array
-    void addGlyphQuad(sf::VertexArray& vertices, sf::Vector2f position, const sf::Color& color, const sf::Glyph& glyph, float italicShear, float outlineThickness = 0)
+    void addGlyphQuad(sf::VertexArray& vertices, const sf::Vector2f& position, const sf::Color& color, const sf::Glyph& glyph, float italicShear, float outlineThickness = 0)
```

Comment:
Please don't change this, it's literally a `double` being copied and makes code more wordy without benefit.

I even think that all the APIs in SFML should use `sf::Vector2f` instead of `const sf::Vector2f&` for shorter IDE suggestions and documentation.

Reply:
@Bromeon fix it.
https://github.com/SFML/SFML/pull/1732/commits/ae2d032d5a33e7f63fb2355b64426eef870eb0e5